addons/mail/static/src/components/activity/activity.xml

```diff
@@ -52,7 +52,7 @@
                         </a>
                     </div>
 
-                    <div class="o_Activity_details" t-if="state.areDetailsVisible">
+                    <div class="o_Activity_details" name="details" t-if="state.areDetailsVisible">
```

Comment:
Unrelated but `t-if` should be on its own tag.

That said, I don't think you need a name here, you can just inherit based on class ex.
`<xpath expr="//*[hasclass('o_NotificationGroup_inlineText')]" position="inside">`